napari/util/__init__.py

```diff
@@ -1 +1,2 @@
 from .misc import is_rgb, segment_normal
+from .temporary_file import temporary_file
```

Comment:
I'm not positive that we want this in our public API...? As opposed to having it be e.g. a testing utility.

Reply:
I'm fine changing it, but what do you have in mind. Leaving file where it is, but dropping this import, or maybe putting it inside of `tests`. We might want to end up using it in multiple places